heading: 0 #347.5695
accel_x_bias: 0 #0.0962
gyro_z_bias: 0 #-0.0000394107
initialized: false